tests/units/infrastructure/repositories/users.repository.test.ts

```
@@ -0,0 +1,67 @@
+  await usersRepository.updateUser('8', {
+    email: 'eight-new@test.com',
+    password: 'password-eight-new',
+  });
+  const newUser = await usersRepository.getUser('8');
+  expect(newUser).toMatchObject({
+    id: '8',
+    email: 'eight-new@test.com',
+  });
+  const passwordMatched = await compare(
+    'password-eight-new',
+    newUser?.passwordHash!
```

ESLint

[error] Optional chain expressions can return undefined by design - using a non-null assertion is unsafe and wrong.

(@typescript-eslint/no-non-null-asserted-optional-chain)